pages/about.html

```diff
@@ -1,12 +1,105 @@
+            </li>
+            <li class="navbar__btn">
+                <a href="" class="button">Resume</a>
+            </li>
+            <li class="navbar__btn">
+                <a href="" class="button">Contact Me</a>
+            </li>
+        </ul>
+    </div>
+</nav>
+
+
+<div class="main">
+  <div class="main__container">
+      <div class="main__content">
+          <h1>Yonese James</h1>
+          <p>Your friendly neighborhood programmer!</p>
+      </div>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Blandit cursus risus at ultrices mi tempus imperdiet nulla malesuada. Sit amet consectetur adipiscing elit pellentesque habitant morbi tristique senectus. Integer vitae justo eget magna. Eros donec ac odio tempor. Ut faucibus pulvinar elementum integer enim. Euismod quis viverra nibh cras pulvinar. Pellentesque sit amet porttitor eget dolor morbi non. Massa tempor nec feugiat nisl pretium fusce id. Ornare quam viverra orci sagittis. Sem fringilla ut morbi tincidunt augue interdum velit euismod in. Imperdiet massa tincidunt nunc pulvinar sapien et ligula ullamcorper.</p>
+  </div>
+</div>
+
+
+<!-- Footer Section -->
+<div class="footer__container">
+    <div class="footer__links">
+        <div class="footer__link--wrapper">
+            <div class="footer__link--items">
+                <h2>About</h2>
+                <a href="">About Me</a>
+            </div>
+            <div class="footer__link--items">
+                <h2>Projects</h2>
+                <a href="">Project 1</a>
+                <a href="">Project 2</a>
+                <a href="">Project 3</a>
+            </div>
+        </div>
+        <div class="footer__link--wrapper">
+            <div class="footer__link--items">
+                <h2>Contact</h2>
+                <a href="">Contact Me</a>
+                <a href="">Resume</a>
+            </div>
+        </div>
+    </div>
+    <div class="social__media">
+        <div class="social__media--wrap">
+            <p class="website__rights">© Yonese 2022. All rights reserved</p>
+            <div class="social__icons">
+                <a href="" class="social__icons--link" target="_blank">
+                    <i class="fab fa-facebook"></i>
+                </a>
+                <a href="" class="social__icons--link" target="_blank">
+                    <i class="fab fa-instagram"></i>
+                </a>
+                <a href="" class="social__icons--link" target="_blank">
+                    <i class="fab fa-twitter"></i>
+                </a>
+                <a href="" class="social__icons--link" target="_blank">
+                    <i class="fab fa-linkedin"></i>
+                </a>
+                <a href="" class="social__icons--link" target="_blank">
+                    <i class="fab fa-youtube"></i>
+                </a>
+            </div>
+        </div>
```

Comment:
We should remove the divs here and replace with more meaningful elements (maybe like a section element with unordered list that has list items that are links to your socials)